bgpd/bgp_evpn_vty.c

```diff
@@ -2878,6 +2900,8 @@ static void evpn_show_route_rd_all_macip(struct vty *vty, struct bgp *bgp,
 				json_rd = NULL;
 			}
 		}
+
+		bgp_dest_unlock_node(dest);
```

Comment:
whilst were here - does the function that provides dest here `*bgp_afi_node_lookup` need to unlock the pdest it looks up?

Reply:
It's already unlocked or I'm missing something? :) evpn_show_route_rd_all_macip is covered as well.